README.md

```diff
@@ -379,37 +382,134 @@ return Single.just(loginModel)
 
 ## Installation
 
+### Cocoapods
+
 Japx is available through [CocoaPods][8]. To install it, simply add the following line to your Podfile:
 
 ```ruby
 pod 'Japx'
 ```
 
-We've added some more functionalites by conforming to Codable for object mapping or Alamofre for networking.
-You can find those convinience extansions here: 
+We've also added some more functionalites like Alamofre or Moya for networking, Rx for reactive programming approach, Objective-C support:
 
 ```ruby
-# Codable
-pod 'Japx/Codable'
-
 # Alamofire
 pod 'Japx/Alamofire'
 
 # Alamofire and RxSwift
 pod 'Japx/RxAlamofire'
 
-# Alamofire and Codable
-pod 'Japx/CodableAlamofire'
+# Moya
+pod 'Japx/Moya'
+
+# Moya and RxSwift
+pod 'Japx/RxMoya'
+
+# Objective-C
+pod 'Japx/ObjC'
+```
+
+Unlike with other dependency managers below, you should always use:
 
-# Alamofire, Codable and RxSwift
-pod 'Japx/RxCodableAlamofire'
+```swift
+import Japx
 ```
 
+regardless of which custom integration you've picked.
+
+**NOTE: Version 4.0.0 of Japx points to Moya `development` branch due to Rx support issues. If you need stable Moya support, please use version 3.0.0. Otherwise point your Podfile to development Moya:**
+
+```ruby
+platform :ios, '10.0'
+use_frameworks!
+
+target 'MyApp' do
+
+  pod 'Moya', :git => "https://github.com/Moya/Moya.git", :branch => "development"
+  pod 'Japx/RxMoya'
+end
+```
+
+### Swift Package Manager
+
+Add the dependency to your `Package.swift` and use in your target
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/infinum/Japx.git", .upToNextMajor(from: "4.0.0"))
+]
+```
+
+Sample `Package.swift`
+
+```swift
+let package = Package(
+    name: "YourDependency",
+    products: [
+        .library(name: "YourDependency", targets: ["YourDependency"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/infinum/Japx.git", .upToNextMajor(from: "4.0.0")),
+    ],
+    targets: [
+        .target(
+            name: "YourDependency",
+            dependencies: [.product(name: "Japx", package: "Japx")]
+        )
+    ]
+)
+```
+
+We've also added some more functionalites like Alamofre or Moya for networking, Rx for reactive programming approach:
```

Comment:
This is duplicated line from above in the Cocoapods example. 
Can we change it to something like:

If you want Alamofire or Moja for networking, Rx for reactive programming approach, here is how to add them/use them